hack/make-rules/test-cmd.sh

```diff
@@ -135,10 +137,6 @@ run_kube_controller_manager
 run_kubelet
 create_node
 SUPPORTED_RESOURCES=("*")
-output_message=$(runTests "SUPPORTED_RESOURCES=${SUPPORTED_RESOURCES[@]}")
-# Ensure that tests were run. We cannot check all resources here. We check a few
-# to catch bugs due to which no tests run.
-kube::test::if_has_string "${output_message}" "Testing kubectl(v1:pods)"
-kube::test::if_has_string "${output_message}" "Testing kubectl(v1:services)"
+runTests "SUPPORTED_RESOURCES=${SUPPORTED_RESOURCES[@]}"
```

Comment:
I'm trying to figure out what this does. I think this is passing a string as $1 to runTests() which is in turn never used for anything. runTests() does not appear to take any arguments.